.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: Plio CI
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  pre-commit:
+    name: Pre-commit
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - uses: pre-commit/action@v2.0.0
+
+  test:
+    name: Test cases
+    runs-on: ubuntu-latest
+    services:
+      # Creates a postgres docker where migrations will run.
+      db:
+        image: postgres:12.3-alpine
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: github_actions
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+      - uses: actions/checkout@v2
+
+      # Install Python. This matches the Python version in Dockerfile.
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
```

Comment:
can't we directly install using the Dockerfile on the new instance?

Reply:
I dont think we can run tests within a container. That's what we are doing at GitHub level in our other deploy_* github actions.

We will need a running instance. Either this or use docker-compose at Github level. I think we can target it separately.
Here it is: #182